src/converse-chatview.js

```diff
@@ -354,6 +354,9 @@
                             'extra_classes': this.getExtraMessageClasses(attrs)
                         })
                     ));
+                    //Where the argument "attrs" come from? Could "attrs" have information from onMessage?
+                    //Could be possible (and would it be ok) to place a button instead of .text(text)?
+                    //Making the button to show the text set here onclick?
```

Comment:
You have to register the event on the relevant Backbone.View.

For example here some event handlers are registered:
https://github.com/jcbrand/converse.js/blob/9df3715ac881b608a88bc9cf3ffd1e7bc6a0f815/src/converse-chatview.js#L91